### DISCUSSION TITLE
[Bug/Help]: README suggestion

### DISCUSSION THREAD

**Discussion by jamess60** [mbentley/docker-timemachine#167] [Help] [answered]
### Describe the Bug

README doesn't make it super obvious that the backup path (default /opt/timemachine) is tied to the TM_USER variable. During my first backup, I filled my docker host root instead of my backup volume because my mount was incorrect. 

### Expected Behavior

README should be tweaked in a way that makes it slightly more apparent that the /opt/xxx backup path mirrors the TM_USER env

### Steps to Reproduce

1. Setup the container with the volume mapped to /opt/timemachine with a none default username
2. Backup

### How You're Launching the Container

```plain
timemachine:
        image: mbentley/timemachine:latest
        container_name: timemachine
        volumes:
            - /timemachine:/opt/netadmin
        environment:
            - TM_USERNAME=netadmin
            - TM_GROUPNAME=timemachine
            - PASSWORD=ITS A SECRET
            - TM_UID=1000
            - TM_GID=1000
            - SET_PERMISSIONS=false
            - VOLUME_SIZE_LIMIT="0"
            - SHARE_NAME=TimeMachine
        restart: always
        privileged: true
        network_mode: "host"
```

### Container Logs

```plain
Not
Applicable
```

### Additional Context

No additional context but awesome image, keep up the good work :)

**Comment by mbentley** [comment, marked as answer]
It's specifically called out [here](https://github.com/mbentley/docker-timemachine#persistent-data-path) but I guess that may need to be moved up higher or something if it isn't obvious.

**Comment by jamess60** [comment]
My bad! I did indeed miss that, apologies! 
Yeah, possibly worth moving it up or making it bigger. 

Thanks again :)

**Comment by mbentley** [comment]
Added a warning note up by the examples to link down further.  Appreciate the feedback!
